### PR TITLE
docs: add more detail steps in workspace preparation

### PR DESCRIPTION
**Pull request by truongnh1992**
Insert steps for updating working directory, create
a new branch for contributing.

Signed-off-by: Nguyen Hai Truong <truongnh@fujitsu.com>

### Ⅱ. Does this pull request fix one issue?

Add more detail steps in workspace preparation

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
This PR for docs

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -91,6 +91,28 @@ upstream   no-pushing (push)
...
+git checkout master
+
+git rebase upstream/master
+```
+
+Create a new branch:
+
+```
+git checkout -b <new-branch>
+```
```
Comment (lowzj) [inline CONTRIBUTING.md]: These commands can be simplified to 
```bash
cd Dragonfly
git checkout -b <new-branch>
git pull --rebase upstream master
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -74,23 +74,45 @@ To put forward a PR, we assume you have registered a GitHub ID. Then you could f
...
-```
+	```
+	$ git remote -v
+	origin     https://github.com/<your-username>/Dragonfly.git (fetch)
+	origin     https://github.com/<your-username>/Dragonfly.git (push)
+	upstream   https://github.com/dragonflyoss/Dragonfly.git (fetch)
+	upstream   no-pushing (push)
+	```
 
 Adding this, we can easily synchronize local branches with upstream branches.
```
Comment (lowzj) [inline CONTRIBUTING.md]: This line should be indented 4 spaces to the right. You can view the formatted doc in your branch [here](https://github.com/truongnh1992/Dragonfly/blob/contribution/CONTRIBUTING.md#workspace-preparation).

Comment (truongnh1992) [reply CONTRIBUTING.md]: Thank @lowzj. I've just updated.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -74,22 +74,44 @@ To put forward a PR, we assume you have registered a GitHub ID. Then you could f
...
+
+	Adding this, we can easily synchronize local branches with upstream branches.
+
+1. **Create a branch** to add a new feature or fix issues
+
+	Update local working directory:
+
+	```
+	cd Dragonfly
+
```
Comment (lowzj) [inline CONTRIBUTING.md]: remove these blank

**Comment by truongnh1992** [comment]
@lowzj I've just resolved all of your comments.

**Pull request merged**